Sources/WalletConnectPairing/PairingClient.swift

```diff
@@ -3,29 +3,60 @@ import WalletConnectUtils
 import WalletConnectRelay
 import WalletConnectNetworking
 import Combine
+import JSONRPC
 
 public class PairingClient: PairingRegisterer {
+
+    public var pingResponsePublisher: AnyPublisher<(String), Never> {
```

Comment:
mb remove it or make internal? As I understand we need this for unit test only

Reply:
No really, I believe ping may be useful